shared/actions/chat.js

```diff
@@ -499,17 +546,25 @@ function * _deleteMessage (action: DeleteMessage): SagaGenerator<any, any> {
 
   if (messageID) {
     // Deleting a server message.
-    // TODO: Use delete message RPC call when it is available
     const clientHeader = yield call(_clientHeader, CommonMessageType.delete, conversationIDKey)
-    clientHeader.supersedes = messageID
+    const conversationState = yield select(_conversationStateSelector, conversationIDKey)
+    let lastMessageID
+    if (conversationState) {
+      const message = conversationState.messages.findLast(m => !!m.messageID)
+      if (message) {
+        lastMessageID = message.messageID
+      }
+    }
```

Comment:
lastMessageID is fine to be 0/ undefined, the backend will do a fallback. also we don't allow falsey ids ever